src/modules/auth/wrappers/with-auth/with-auth.tsx

```diff
@@ -62,7 +60,6 @@ export const withAuth = (
 					await toHome();
 				} else {
 					// User is logged out, but the page can be seen without logging in
-					setShowPage(true);
 				}
```

Comment:
Intentional empty `else`?

Reply:
yes, the comment describe why nothing should happen here
the minifier will probably remove the empty else